test/topology/util.py

```diff
@@ -91,4 +92,5 @@ async def token_ring_matches():
                                f" according to {srv}, symmetric difference: {diff}")
                 return None
             return True
-        await wait_for(token_ring_matches, deadline, period=.5)
+        if not token_ring_matches:
```

Comment:
I don't understand this change. Isn't `not token_ring_matches` always False, because token_ring_matches is bound to a function. Did you mean to invoke it? 

Should this be done inside wait_for()?

Reply:
Goos catch, I omitted the call operator.  Will fix.
From what I saw,  `wait_for` checks the deadline before calling the function so this should ensure that `token_ring_matches` is called once unconditionally.

Reply:
Changed that in `wait_for` in the next version